src/main/kotlin/org/rust/lang/core/completion/RsCommonCompletionProvider.kt

```diff
@@ -227,20 +228,6 @@ object RsCommonCompletionProvider : RsCompletionProvider() {
     override val elementPattern: ElementPattern<PsiElement>
         get() = PlatformPatterns.psiElement().withParent(psiElement<RsReferenceElement>())
 
-    private val simplePathPattern: ElementPattern<PsiElement>
-        get() {
-            val simplePath = psiElement<RsPath>()
-                .with(object : PatternCondition<RsPath>("SimplePath") {
```

Comment:
looks like `PatternCondition` is unused import in this file now

Reply:
Fixed